src/lib/components/auth/Auth0Button.svelte

```
@@ -1,4 +1,5 @@
 <script lang="ts">
+  import { WEBUI_BASE_URL } from '$lib/constants';
```

ESLint

[error] 'WEBUI_BASE_URL' is defined but never used.

(@typescript-eslint/no-unused-vars)